JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.ts

```diff
@@ -32,88 +43,75 @@ export class AppComponent {
     });
   }
 
-  onOptionChanged(e: any) {
+  onOptionChanged = (e: any) => {
```

Comment:
Minor comment.

Can we improve typing here and replace an ```any``` with a strict type?

Reply:
Changed all any in this file to scheduler event types